### PR TITLE
Add testUrlWithGet flag

### PR DESCRIPTION
**Pull request by maoznir**

### CODE REVIEW

File: test/api.test.js [hunk 1]
```diff
@@ -1,97 +1,100 @@
-describe('API player tests', () => {
-  beforeEach(async () => {
-    await page.setViewport({width: 1280, height: 1800});
-    await page.goto('http://localhost:3000/api.html', {waitUntil: 'load'});
-    await page.evaluate(() => {
-      Object.defineProperty(HTMLMediaElement.prototype, 'playing', {
-        get: function() {
-          return !!(this.currentTime > 0 && !this.paused && !this.ended &&
+const urls = ['http://localhost:3000/api.html', 'http://localhost:3000/api.html?testUrlWithGet=true'];
```
Comment (asisayag2) [inline test/api.test.js]: @maoznir , come to think about it, why not having a param of testUrlWith=get/head?

File: test/api.test.js [hunk 2]
```diff
@@ -1,97 +1,100 @@
...
-  });
 
-/*  it('Test fullscreen', async () => {
-    await page.waitFor(1000);
-    expect(await page.evaluate(() => document.webkitIsFullScreen)).toBeFalsy();
-    await page.click('#vid-maximize');
-    await page.waitFor(1000);
-    expect(await page.evaluate(() => document.webkitIsFullScreen)).toBeTruthy();
-  });*/
+    /*  it('Test fullscreen', async () => {
```
Comment (asisayag2) [inline test/api.test.js]: @maoznir why is that commented out?

**Comment by yaniv-li** [comment]
@maoznir I would need a better explanation

File: docs/api.html [hunk 3]
```diff
@@ -22,8 +22,9 @@
     window.addEventListener('load', function(){
 
       var cld = cloudinary.Cloudinary.new({ cloud_name: 'demo' });
+      var options = location.search.split('testUrlWithGet=')[1] ? {testUrlWithGet : true} : {};
```
Comment (asisayag2) [inline docs/api.html]: @maoznir, on second thought, I would change the queryParam to - testUrlWith=get/head

File: src/video-player.js [hunk 4]
```diff
@@ -44,6 +44,15 @@ const PLAYER_PARAMS = CLOUDINARY_PARAMS.concat([
   'textTracks'
 ]);
 
+const TEST_URL_DEFAULT_REQUEST = { method: 'head' };
+const TEST_URL_GET_REQUEST = { method: 'get', withCredentials: true, headers: { Range: 'bytes=0-0' } };
+
+const getTestUrlRequest = (uri, options = {}) => {
```
Comment (asisayag2) [inline src/video-player.js]: I would send into this method only the relevant option (options.testUrlMethod).

File: test/api.test.js (hunk 2 above)
Comment (maoznir) [reply test/api.test.js]: Noted but not related to this pr - was commented out in master

File: docs/api.html (hunk 3 above)
Comment (maoznir) [reply docs/api.html]: Ok

File: src/video-player.js (hunk 4 above)
Comment (maoznir) [reply src/video-player.js]: Good idea

Comment (maoznir) [reply src/video-player.js]: done

File: docs/api.html (hunk 3 above)
Comment (maoznir) [reply docs/api.html]: done

**Pull request merged**